packages/contracts-bedrock/test/L2/NativeAssetLiquidity.t.sol

```diff
@@ -0,0 +1,184 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Testing utilities
+import { CommonTest } from "test/setup/CommonTest.sol";
+
+// Error imports
+import { Unauthorized, InvalidAmount } from "src/libraries/errors/CommonErrors.sol";
+
+/// @title NativeAssetLiquidity_TestInit
+/// @notice Reusable test initialization for `NativeAssetLiquidity` tests.
+contract NativeAssetLiquidity_TestInit is CommonTest {
+    /// @notice Emitted when an address withdraws native asset liquidity.
+    event LiquidityWithdrawn(address indexed caller, uint256 value);
+
+    /// @notice Emitted when an address deposits native asset liquidity.
+    event LiquidityDeposited(address indexed caller, uint256 value);
+
+    /// @notice Emitted when an address funds the contract.
+    event LiquidityFunded(address indexed funder, uint256 value);
+
+    /// @notice Test setup.
+    function setUp() public virtual override {
+        enableCustomGasToken();
+        super.setUp();
+    }
+
+    /// @notice Tests that contract is set up correctly.
+    function test_setup_succeeds() public view {
```

Comment:
Same nit as above, prefer not to test anything in the test initializer contract

Reply:
Also for version tests throughout the PR, please just check that it returns a non-empty string so we can avoid changing this test every time the version changes in the contract

Reply:
Future note to self to have "is valid semver" check somewhere